scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
```

Comment:
:stop_sign: **Security Vulnerability**: This file contains multiple security vulnerabilities, including hardcoded credentials, command injection, path traversal, and SQL injection. These should be removed immediately if they are not intended for testing purposes.[^1][^2][^3][^4]
```suggestion
// Remove or securely store credentials
// const apiKey = process.env.API_KEY
// const password = process.env.PASSWORD
// const dbConnection = process.env.DB_CONNECTION

// Fix command injection
function executeCommand(userInput: string) {
    // Use a whitelist of allowed commands or sanitize input
    const allowedCommands = ['ls', 'dir']
    if (allowedCommands.includes(userInput)) {
        child_process.exec(userInput)
    } else {
        throw new Error('Invalid command')
    }
}

// Fix path traversal
function readUserFile(filename: string) {
    const safePath = path.normalize(path.join('/tmp', filename))
    if (safePath.startsWith('/tmp/')) {
        return fs.readFileSync(safePath)
    }
    throw new Error('Invalid file path')
}

// Fix SQL injection
const query = 'SELECT * FROM users WHERE id = ?'
// Use parameterized queries with proper escaping
```